src/runtime_src/driver/xclng/drm/xocl/userpf/xocl_ioctl.c

```diff
@@ -35,27 +35,23 @@ xuid_t uuid_null = NULL_UUID_LE;
 #endif
 
 static int
-xocl_client_lock_bitstream(struct xocl_dev *xdev, struct client_ctx *client)
-{
-	if (atomic_read(&client->xclbin_locked))
+xocl_client_lock_bitstream_helper(struct xocl_dev *xdev, struct client_ctx *client) {
```

Comment:
_helper() -> _nolock() to better align the intention of this function?

Reply:
Yes.